rosidl_pycommon/rosidl_pycommon/__init__.py

```diff
@@ -70,7 +70,9 @@ def generate_files(
     ros_interface_files = {}
     for ros_interface_file in args.get('ros_interface_files',  []):
         p = pathlib.Path(ros_interface_file)
-        ros_interface_files[p.stem] = p
+        # e.g. ('Empty', 'msg')
+        key = (p.stem, p.suffix[1:])
```

Comment:
This is very nit-picky, but we are used to thinking about things in terms of `pkg/type/name`.  So this tuple being (`name`/`type`) seems a little odd.  Should we invert the order?  If we do this, we'll also have to fix `type_source_key` below.

Reply:
No problem, I'll invert it. I had written it both ways and just gone this way, but no strong opinion on it